tests/bootstrap.php

```diff
@@ -1,8 +1,17 @@
 <?php
+
```

Comment:
Is `npm run lint:php` working in your local?

There's an error in mine:

```sh
PHP Fatal error:  Cannot declare class XWP\BlockScaffolding\TestCase, because the name is already in use in /Users/ryankienstra/Projects/sites/sitio/wp-content/plugins/AMP-gutenberg-dynamic-block/tests/php/TestRouter.php on line 16
```